Bottom: 498152a704d464975f8918850896c5dd0a3d43d0
Top:    d5027a8128e786e4e0a63fdd49f5c80daf616db0
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-29 19:17:12 -0400

Refresh of spl-bss

---

diff --git a/board/ElecArts/ea3131/Makefile b/board/ea/ea3131/Makefile
similarity index 100%
rename from board/ElecArts/ea3131/Makefile
rename to board/ea/ea3131/Makefile
diff --git a/board/ElecArts/ea3131/config.mk b/board/ea/ea3131/config.mk
similarity index 100%
rename from board/ElecArts/ea3131/config.mk
rename to board/ea/ea3131/config.mk
diff --git a/board/ElecArts/ea3131/ea3131.c b/board/ea/ea3131/ea3131.c
similarity index 100%
rename from board/ElecArts/ea3131/ea3131.c
rename to board/ea/ea3131/ea3131.c
diff --git a/board/ElecArts/ea3131/lowlevel_init.S b/board/ea/ea3131/lowlevel_init.S
similarity index 100%
rename from board/ElecArts/ea3131/lowlevel_init.S
rename to board/ea/ea3131/lowlevel_init.S
diff --git a/boards.cfg b/boards.cfg
index 8342c43..58adfd5 100644
--- a/boards.cfg
+++ b/boards.cfg
@@ -153,8 +153,8 @@ openrd_ultimate              arm         arm926ejs   openrd              Marvell
 rd6281a                      arm         arm926ejs   -                   Marvell        kirkwood
 sheevaplug                   arm         arm926ejs   -                   Marvell        kirkwood
 dockstar                     arm         arm926ejs   -                   Seagate        kirkwood
-ea3131                       arm         arm926ejs   ea3131              ElecArts       lpc313x
-ea3152                       arm         arm926ejs   ea3131              ElecArts       lpc313x
+ea3131                       arm         arm926ejs   ea3131              ea             lpc313x
+ea3152                       arm         arm926ejs   ea3131              ea             lpc313x
 rw_egg                       arm         arm926ejs   rw_egg              Redwire        lpc313x
 jadecpu                      arm         arm926ejs   jadecpu             syteco         mb86r0x
 mx25pdk                      arm         arm926ejs   mx25pdk             freescale      mx25		mx25pdk:IMX_CONFIG=board/freescale/mx25pdk/imximage.cfg
diff --git a/include/configs/ea3131.h b/include/configs/ea3131.h
index 651210f..6d641b4 100644
--- a/include/configs/ea3131.h
+++ b/include/configs/ea3131.h
@@ -171,9 +171,9 @@
 "dt_addr=0x32000000\0" \
 "rd_addr=0x32100000\0" \
 "usbtty=cdc_acm\0" \
-"ramargs=setenv bootargs console=ttyS0,115200n8 root=/dev/mmcblk0p3 rootwait init=/etc/preinit rw loglevel=7\0" \
+"ramargs=setenv bootargs console=ttyS0,115200n8 root=/dev/mmcblk0p3 rootwait init=/etc/preinit rw loglevel=8\0" \
 "serverip=192.168.1.48\0" \
-"nfsargs=setenv bootargs console=ttyS0,115200n8 root=/dev/nfs init=/etc/preinit rw nfsroot=${serverip}:${rootpath} ip=dhcp loglevel=7\0" \
+"nfsargs=setenv bootargs console=ttyS0,115200n8 root=/dev/nfs init=/etc/preinit rw nfsroot=${serverip}:${rootpath} ip=dhcp loglevel=8\0" \
 "rootpath=/tftpboot/arm\0" \
 "bootfile=uImage\0" \
 "ramfile=rootfs.ext2.gz.uboot\0" \
